src/Tweaks/Build/Options.cs

```diff
@@ -17,5 +17,22 @@ internal partial class Options : BaseOptionModel<Options>
         [Description("Specifies the obj and bin folders should be deleted when running the Clean command.")]
         [DefaultValue(true)]
         public bool DeleteOuputArtifactsOnClean { get; set; } = true;
+
+        /// <summary>   If Enabled, the Build Output Windows will output the reason why a project is
+        ///             being built.</summary>
+        /// <remarks>   Ultimately, this will set the DWORD registry value <c>U2DCheckVerbosity</c> in the 
+        ///             Visual Studio registry hive under the <c>General</c> key. DO NOT alter the
+        ///             property name as it reflects the registry value that is set. Note the
+        ///             <see cref="OverrideCollectionNameAttribute"/> and <see cref="OverrideDataTypeAttribute"/>
+        ///             have been applied so the value is loaded/saved in the appropriate place and is of
+        ///             the appropriate type. <a href="https://github.com/madskristensen/Tweakster/issues/60">
+        ///             Issue documentation</a> </remarks>
+        [Category(_build)]
+        [DisplayName("Up-To-Date Check Verbose")]
+        [Description("If Enabled, the Build Output Windows will output the reason why a project is being built.")]
+        [DefaultValue(false)]
+        [OverrideCollectionName("General")]
+        [OverrideDataType(SettingDataType.Bool)]
```

Comment:
What does it solve to add all this to the options?

Reply:
I'm not sure I understand the question. This Tweak is used to change a Visual Studio option hidden in the private registry hive that has no GUI setting [see comment here](https://developercommunity.visualstudio.com/t/full-build-every-time-with-155/161997#T-N162336). It made sense to add it to the Build tweaks since it fits in there nicely with the other options.

Reply:
Ahh, I missed that part. Thanks for clarifying